otter/models/intents.py

```diff
@@ -14,3 +19,35 @@ class ModifyGroupState(object):
 def perform_modify_group_state(dispatcher, mgs_intent):
     """Perform a :obj:`ModifyGroupState`."""
     return mgs_intent.scaling_group.modify_state(mgs_intent.modifier)
+
+
+@attributes(['tenant_id', 'group_id'])
+class GetScalingGroupInfo(object):
+    """Look up a scaling group, along with its `state` and launch config."""
+
+
+@deferred_performer
+@inlineCallbacks
+def perform_get_scaling_group_info(log, store, dispatcher, intent):
+    """
+    Perform :obj:`GetScalingGroupInfo`.
+
+    :param log: bound log
+    :param IScalingGroupCollection store: collection of groups
+    :param dispatcher: dispatcher provided by perform
+    :param GetScalingGroupInfo intent: the intent
+    """
+    group = yield store.get_scaling_group(
+        log, intent.tenant_id, intent.group_id)
+    state = yield group.view_state()
```

Comment:
Non-blocking:  since `view_manifest` got changed to deal with force-deleting groups (so you can return the state, the config, and the launch config), it can probably be used here to make this a single cass call instead of 2.  That may or may not be a pointless optimization.